pydocx/openxml/wordprocessing/run.py

```diff
@@ -29,18 +17,19 @@ class Run(XmlModel):
     properties = XmlChild(type=RunProperties)
 
     children = XmlCollection(
-        EmbeddedObject,
-        TabChar,
-        Break,
-        NoBreakHyphen,
-        Text,
-        Drawing,
-        Picture,
-        DeletedText,
-        FootnoteReference,
-        FootnoteReferenceMark,
-        FieldChar,
-        FieldCode,
+        'wordprocessing.EmbeddedObject',
+        'wordprocessing.TabChar',
+        'wordprocessing.Break',
+        'wordprocessing.NoBreakHyphen',
+        'wordprocessing.Text',
+        'wordprocessing.Drawing',
+        'wordprocessing.Picture',
+        'wordprocessing.DeletedText',
+        'wordprocessing.FootnoteReference',
+        'wordprocessing.FootnoteReferenceMark',
+        'wordprocessing.FieldChar',
+        'wordprocessing.FieldCode',
+        'markup_compatibility.AlternateContent',
```

Comment:
I don't think testcase coverage is sufficient to mitigate that concern. We're not the only users of PyDocx. We also need to think about the developer experience for other folks that will be using and extending the library.